extensions/notebook/src/dialog/configurePython/configurePythonWizard.ts

```diff
@@ -67,7 +67,7 @@ export class ConfigurePythonWizard {
 		} else {
 			wizardTitle = localize('configurePython.wizardNameWithoutKernel', 'Configure Python to run kernels');
 		}
-		this._wizard = azdata.window.createWizard(wizardTitle);
+		this._wizard = azdata.window.createWizard(wizardTitle, 600);
 		let page0 = azdata.window.createWizardPage(localize('configurePython.page0Name', 'Configure Python Runtime'));
```

Comment:
well, just realized we are using single quote for the resource strings in this file. while you are on this file, do you mid updating the resource strings to use double quote?

Reply:
Sure